routes/leaguesearch.js

```diff
@@ -48,6 +48,10 @@ router.get('/:league', function(req, res) {
 					"league": "$league"
 				},
 				"quizCount": { "$sum": 1}
+				/*
+				,
```

Comment:
can you just remove this block? I have a feeling that this is gonna make a merge conflict